lib/foreman_remote_execution/engine.rb

```diff
@@ -123,6 +123,9 @@ class Engine < ::Rails::Engine
           ctx.permit :execution
         end
 
+        extend_template_helpers ForemanRemoteExecution::RendererMethods
+        allowed_template_helpers :load_job_invocations
```

Comment:
do we also need `load_job_invocation_by_id`? and do we need to use `allowed_template_helpers` if we use `extend_template_helpers` (I don't recall but I though the first is enough)?